core/src/components/menu/test/multiple/menu.e2e.ts

```diff
@@ -0,0 +1,74 @@
+import { expect } from '@playwright/test';
+import { configs, test } from '@utils/test/playwright';
+
+/**
+ * This behavior does not vary across modes/directions
+ */
+configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) => {
+  test.describe(title('menu: multiple'), () => {
```

Comment:
Please include an annotation for the issue. Not sure if it can go on the `describe` level or it should be inside the `beforeEach`.

Reply:
Thanks for the reminder! Added in https://github.com/ionic-team/ionic-framework/pull/28269/commits/55704489ea3ac57fa099d52d43f71e33f81b797c and https://github.com/ionic-team/ionic-framework/pull/28269/commits/90425eb61f8f3e916b977cb75d9037b468b6af28